app/src/main/java/chat/rocket/android/chatroom/presentation/ChatRoomPresenter.kt

```diff
@@ -184,7 +184,8 @@ class ChatRoomPresenter @Inject constructor(
                             isBroadcast = chatIsBroadcast, isRoom = true
                         )
                     )
-                    if (oldMessages.isNotEmpty()) {
+                    val lastSyncDate = messagesRepository.getLastSyncDate()
```

Comment:
This value should be per roomId, otherwise we will use the last sync date from other rooms

Reply:
@luciofm yep, it's my mistake. I've tested on my local branch with one room only.
Gonna fix it

Reply:
fixed